src/camera.cpp

```diff
@@ -467,7 +466,6 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 busytime,
 	const bool swimming = (movement_XZ || player->swimming_vertical) && player->in_liquid;
 	const bool climbing = movement_Y && player->is_climbing;
 	if ((walking || swimming || climbing) &&
-			m_cache_view_bobbing &&
```

Comment:
It would be better to replace this with '&& view_bobbing_amount != 0.0' so that the view bobbing code is not activated at all when it is not wanted, this is the advantage of the bool.